### PR TITLE
Remove path from "include currency-info.php" in locale-info.php

### PR DESCRIPTION
**Pull request by Konamiman**
### All Submissions:

* [ ] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
* [ ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This pull request is the same as https://github.com/woocommerce/woocommerce/pull/30935, but targeting the `release/5.9` branch for inclusion in WooCommerce 5.9. Same testing instructions and changelog entry.

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### FOR PR REVIEWER ONLY:

* [x] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: i18n/locale-info.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 defined( 'ABSPATH' ) || exit;
 
-$locales = include WC()->plugin_path() . '/i18n/currency-info.php';
+$locales = 'currency-info.php';
```
Comment (peterfabian) [inline i18n/locale-info.php]: it's missing the `include` portion :)

**Review by peterfabian — CHANGES_REQUESTED** [review]
Please update. Thanks!

File: i18n/locale-info.php (hunk 1 above)
Comment (Konamiman) [reply i18n/locale-info.php]: 🤦‍♂️

**Review by peterfabian — APPROVED** [review]
Looks good now, thanks!

**Comment by Konamiman** [comment]
Merged into `release/5.9` for inclusion in WooCommerce 5.9 RC 2. Needs to be cherry-picked into `trunk`.

**Comment by Konamiman** [comment]
No need to cherrypick to `trunk` as the change has been already applied there.

**Pull request merged**